mrblib/haconiwa/base.rb

```diff
@@ -568,6 +568,9 @@ class Capabilities
       cap_dac_override
       cap_fowner
       cap_fsetid
+      cap_net_bind_service
+      cap_net_broadcast
```

Comment:
Ah, but , man says that `cap_net_broadcast` is not used for now. 
http://man7.org/linux/man-pages/man7/capabilities.7.html